Snowflake.Data/Logger/SFLoggerFactory.cs

```diff
@@ -8,10 +8,52 @@ namespace Snowflake.Data.Log
 {
     class SFLoggerFactory
     {
+        private static bool isLoggerEnabled = true;
+
+        private static SFLogger logger = null;
+
+        private SFLoggerFactory()
+        {
+        }
+
+        public static void disableLogger()

```

Comment:
I am not an expert on .net scoping. Have you verified that if this method is accessible from a different project?

Reply:
Yes, confirmed that the static methods are accessible when called from a different project.
The attached "PlugAndPlayEx" file can be placed in the Snowflake.Data.Tests project for testing as well